kong-0.5.0-1.rockspec

```diff
@@ -29,7 +29,8 @@ dependencies = {
 
   "luasocket ~> 2.0.2-5",
   "lrexlib-pcre ~> 2.7.2-1",
-  "lua-llthreads2 ~> 0.1.3-1"
+  "lua-llthreads2 ~> 0.1.3-1",
+  "sha1 ~> 0.5-1"
```

Comment:
No, sha1 is a fallback for unit tests. Otherwise we use resty.string because it is included in OpenResty. However am considering luacrypto for the JWT plugin, would that be of any use for the HMAC? I suppose so.